# request to take off one/all drones
int8 ACTIVE = -1
int8 ALL = -2

int8 id
---
bool status